op-chain-ops/cmd/celo-migrate/state.go

```diff
@@ -62,38 +65,6 @@ func applyStateMigrationChanges(genesis *core.Genesis, dbPath string, commit boo
 		return nil, fmt.Errorf("cannot open StateDB: %w", err)
 	}
 
-	// So far we applied changes in the memory VM and collected changes in the genesis struct
-	// Now we iterate through all accounts that have been written there and set them inside the statedb.
-	// This will change the state root
-	// Another property is that the total balance changes must be 0
```

Comment:
Why have we deleted this? I thought this was the meat of what the state migration script was supposed to be doing